samples/iothub-connect-to-eventhubs/README.md

```diff
@@ -0,0 +1,39 @@
+
+A common pattern is for users of IoT Hub to consume information from an Event Hub instance that is provisioned and owned by the IoT Hub service. The [IoT Hub documentation](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-endpoints) refers to this Event Hub instance as the "built-in Event Hub-compatible endpoint that IoT Hub exposes", making it an important part of the IoT Hub story.  One of the features supported by IoT Hub is a fail-over scenario, which may be initiated by customers, and which is likely to result in IoT Hub changing the Event Hub instance to which it publishes events. Because this Event Hub instance association is owned by IoT Hub, developers cannot assume a static connection string to access it, but instead must query the IoT Hub service for the active instance.
+
+This sample illustrates querying the IoT Hub service to obtain the connection string of the Event Hub instance to which IoT Hub is currently publishing as its "built-in" endpoint.
+
+## Getting started
+
+- **Microsoft Azure Subscription:**  To use Azure services, including Azure Event Hubs, you'll need a subscription.  If you do not have an existing Azure account, you may sign up for a free trial or use your MSDN subscriber benefits when you [create an account](https://account.windowsazure.com/Home/Index).
+
+- **.NET Core 3.1:** This sample targets [.NET Core 3.1](https://docs.microsoft.com/en-us/dotnet/standard/frameworks#how-to-specify-target-frameworks) and makes use of language features that were introduced in C# 8.0.  In order to build and run the sample without modifications, you will need the [.NET Core SDK](https://dotnet.microsoft.com/download) available.  If you are using Visual Studio, versions prior to Visual Studio 2019 are not compatible with the tools needed to build C# 8.0 projects nor target .NET Core 3.1.  Visual Studio 2019, including the free Community edition, can be downloaded [here](https://visualstudio.microsoft.com/vs/).
+
+- **IoT Hub Instance:** To interact with Azure IoT Hub, you'll also need to have an IoT Hub instance available.  If you are not familiar with creating Azure resources, you may wish to follow the step-by-step guide for [creating an IoT hub using the Azure portal](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-through-portal).  There, you can also find detailed instructions for using the [Azure CLI](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-using-cli), [Azure PowerShell](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-create-using-powershell), or [Azure Resource Manager (ARM) templates](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-rm-template-powershell) to create an IoT Hub.
```

Comment:
IoT Hub

Reply:
I believe this is in reference to the title (bold area) on line 27.   I've taken the suggestion and removed the word "Instance".  Please let me know if I interpreted this incorrectly.